portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -1101,17 +1085,7 @@ definition {
 				panel = "Applications",
 				portlet = "Content Dashboard");
 
-			AssertElementPresent(locator1 = "//span[contains(@title, 'This content has a display page')]");
-		}
-
-		task ("Delete WC") {
-			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
-
-			WebContent.viewTitle(webContentTitle = "Webcontent Title");
-
-			WebContent.deleteCP(
-				siteURLKey = "test-site-name",
-				webContentTitle = "Webcontent Title");
+			AssertElementPresent(locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED_DISPLAY_PAGE");
```

Comment:
In the test _ShouldExhibitWebContentNotAssociatedWithDisplayPageTemplate_ , you assert both the WC name is present and the Display Page icon is not present, but in this test you are just asserting the Display Page icon present. Why don't you assert the WC name in this test also?